src/celestia/qt/qttimetoolbar.cpp

```diff
@@ -68,26 +68,49 @@ TimeToolBar::TimeToolBar(CelestiaCore* _appCore,
     connect(halfTimeAction, SIGNAL(triggered()), this, SLOT(slotHalfTime()));
     addAction(halfTimeAction);
 
-    connect(pauseAction, SIGNAL(triggered()), this, SLOT(slotPauseTime()));
-    addAction(pauseAction);
-
-    connect(realTimeAction, SIGNAL(triggered()), this, SLOT(slotRealTime()));
-    addAction(realTimeAction);
+    pauseResumeBtn = new QToolButton(this);
+    connect(pauseResumeBtn, SIGNAL(clicked()), this, SLOT(slotPauseResumeTime()));
+    if (textButton) {
```

Comment:
Please use codestyle adopted in Celestia:
```
if ()
{
}
else
{
}
```

Reply:
Will modify.  I did look at codestyle and matched it.  I must have gotten some older code by accident.  Sorry.